mesh/notifications/models.py

```diff
@@ -1,7 +1,11 @@
 from django.db import models
+from mesh.accounts.models import Account
 
 # Create your models here.
 
 class Notification(models.Model):
-    #TODO
-    pass
\ No newline at end of file
+    accountID = models.ForeignKey(Account, on_delete=models.CASCADE)
+    notificationID = models.AutoField(primary_key=True)
+    timeStamp = models.TimeField()
+    notificationContextJSON = models.JSONField()
```

Comment:
what is notficationContext supposed to be about?

Reply:
I followed what was on the table but I think its just information about the notification like the message/type of notification?